sdk/cosmos/azure-cosmos/samples/document_management_async.py

```diff
@@ -85,25 +85,52 @@ async def read_items(container):
 async def query_items(container, doc_id):
     print('\n1.4 Querying for an  Item by Id\n')
 
-    # enable_cross_partition_query should be set to True as the container is partitioned
+    # The enable_cross_partition_query option is not used by the async client - this is the default behavior
     # In this case, we do have to await the asynchronous iterator object since logic
     # within the query_items() method makes network calls to verify the partition key
     # definition in the container
     query_items_response = container.query_items(
         query="SELECT * FROM r WHERE r.id=@id",
         parameters=[
-            { "name":"@id", "value": doc_id }
-        ],
-        enable_cross_partition_query=True
+            {"name": "@id", "value": doc_id}
+        ]
     )
 
     items = [item async for item in query_items_response]
 
     print('Item queried by Id {0}'.format(items[0].get("id")))
 
 
+async def query_items_with_continuation_token(container):
+    print('\n1.5 Querying for Items using Pagination and Continuation Token\n')
+
+    # When using Pagination, max_item_count will limit the number of items in each page
+    query_iterable = container.query_items(
+        query="SELECT * FROM r",
+        max_item_count=1
+    )
+
+    item_pages = query_iterable.by_page()
+
+    # It is worth mentioning that from Python 3.10, you can use the syntax below to iterate as opposed to __anext__()
+    # first_page = await anext(item_pages)  # cspell:disable-line
+
+    first_page = await item_pages.__anext__()
+    continuation_token = item_pages.continuation_token
+    second_page_items = [item async for item in await item_pages.__anext__()]
+
+    # Now we use the continuation token from the first page to immediately access the second page and compare items
+    items_from_continuation = query_iterable.by_page(continuation_token)
+    second_page_items_with_continuation = [item async for item in await item_pages.__anext__()]
+
+    assert second_page_items == second_page_items_with_continuation
```

Comment:
We shouldn't have `assert` statements in a sample.
This sample feels a bit test-like - are we able to frame this in terms of the context of _why_ I would need to use continuation tokens between pages like this?

Reply:
I was just trying to come up with a way to show users that the results would be the same in the second page - I can remove the assertion though.

The use case for continuation tokens is to load only x number of items each time we iterate as opposed to going through the whole result set at once.